src/index.js

```diff
@@ -3,4 +3,4 @@ import UI from './modules/UI.js';
 
 document.addEventListener('DOMContentLoaded', UI.showScores);
 
-document.querySelector('.leaderForm').addEventListener('submit', ((code) => UI.addScore(code)));
\ No newline at end of file
+document.querySelector('.submit').addEventListener('click', ((code) => UI.addScore(code)));
```

Comment:
- [ ] Because you are targeting the submit button nothing happens when I click on it you should add an event listener to the form here [how to do it here ](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/submit_event) you should when clicking on submit the API get called 
and also as previous reviews asked when clicking on the refresh  a function should fetch the API data and display them  in order to match the requirements and also have a working app